src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVFileUtils.java

```diff
@@ -0,0 +1,184 @@
+package org.broadinstitute.hellbender.tools.spark.sv.utils;
+
+import htsjdk.samtools.*;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.spark.utils.HopscotchSet;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+
+import java.io.*;
+import java.util.*;
+
+public final class SVFileUtils {
+
+    private static final String REFERENCE_GAP_INTERVAL_FILE_COMMENT_LINE_PROMPT = "#";
+
+    /**
+     * Creates a directory, in local FS, HDFS, or Google buckets to write to.
+     */
+    public static boolean createDirToWriteTo(final String pathString) {
```

Comment:
What would you suggest for the name?
I am creating a dir to write some (SAM) files in a prototyping tool.

Reply:
```createDirectory```? I just don't why would you create a directory if you are not going to write into it, you or something else later on so that part of the name is rather pointless.... That said it seems this method would better belong to BucketUtils pairing with ```createFile``` as I say below.

Reply:
renamed.